content/blogs/2022-02-01-kestra-opensource.md

```diff
@@ -81,7 +81,7 @@ jq -r '.name' /tmp/query.json
     WHERE shippedDate = '{{ now() }}'
     AND shippedCountry = 'FR'
 - id: "return"
-  type: "io.kestra.core.tasks.debugs.Return"
+  type: "io.kestra.plugin.core.debug.Return"
```

Comment:
we deliberately didn't rename it in old blog posts to keep the state as it was (imagine someone still on old version), but no strong opinion, we can rename everywhere ✅